app/presenters/activity_presenter.rb

```diff
@@ -56,11 +56,6 @@ def flow
     I18n.t("activity.flow.#{super}")
   end
 
-  def finance
```

Comment:
I'm not sure this counts as 'deleting from spec files' so it might have snuck into this commit by accident?

Reply:
I'll move it. Thanks

Reply:
I've moved this to the right commit. Thanks for pointing that out.